sdcm/nemesis.py

```diff
@@ -133,6 +153,15 @@ def report(self):
         for operation in self.operation_log:
             self.log.info(operation)
 
+    def get_subclasses_disrupt_methods(self, subclasses_list):
+        disrupt_methods_list = []
+        for subclass in subclasses_list:
+            method_name = re.search('self\.(?P<method_name>disrupt_[A-Za-z_]+?)\(.*\)', inspect.getsource(subclass), flags=re.MULTILINE)
```

Comment:
???

Reply:
Because the all methods are defined in Base Nemesis class, and not in Each Monkey class, i need to find in each monkey class, which method is used. One way to use the Monkey name and correlate it to method name, but some Class names and methods name are not the same.
So i decide to catch the name of method from source.